game.py

```diff
@@ -64,19 +66,29 @@ def put_down_card_on_table(self, player, card):
 
     def try_playing_matching_number_or_color_card(self, player):
```

Comment:
I got confused little bit by the name, it isn't right to say that it tries to plan a color card, for example, the stop card has a color, and the function doesn't play that. 

consider having anther name :) (I can help if you want)